src/verify_cert.rs

```diff
@@ -912,6 +931,7 @@ mod tests {
         assert_eq!(build_linear_chain(7), Err(Error::MaximumPathDepthExceeded));
     }
 
+    #[cfg(feature = "alloc")]
```

Comment:
These should probably go in the previous commit that introduces them?

Reply:
Fixed. Good catch.